RetPageOriginDockerfile

```diff
@@ -0,0 +1,30 @@
+###
+# this dockerfile produces image/container that serves customly packaged hubs and admin static files
+# the result container should serve reticulum as "hubs_page_origin" and "admin_page_origin" on (path) "/hubs/pages"
+###
+from node:16.13 as builder
+run mkdir -p /hubs/admin/ && cd /hubs
+copy package*.json ./
+copy admin/package*.json admin/
```

Comment:
nit: Since we only seek to copy `package.json` and `package-lock.json`, maybe these should list them explicitly instead of catching them with a wildcard:
```
copy package.json ./
copy package-lock.json ./
copy admin/package.json admin/
copy admin/package-lock.json admin/
```

Reply:
yep will do